src/gt4py/next/program_processors/runners/dace_fieldview/gtir_dataflow.py

```diff
@@ -487,64 +523,156 @@ def _visit_neighbors(self, node: gtir.FunCall) -> DataExpr:
 
         offset_dim = gtx_common.Dimension(offset, kind=gtx_common.DimensionKind.LOCAL)
         neighbor_idx = dace_gtir_utils.get_map_variable(offset_dim)
-        me, mx = self._add_map(
-            f"{offset}_neighbors",
-            {
-                neighbor_idx: f"0:{offset_provider.max_neighbors}",
-            },
-        )
+
         index_connector = "__index"
+        op_code = f"__field[{index_connector}]"
```

Comment:
I do not like `op_code` but it is consistent with what is used elsewhere.
I would suggest you to call it `tasklet_expression` ([justification](https://en.cppreference.com/w/cpp/language/expressions)).
My main point is that "tasklet" should be included in the name.

This applies to all other occurrences of `op_code`.

Reply:
Good idea, agreed.